src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -712,6 +713,7 @@ class UnifiedShareForm extends React.Component<USFProps, State> {
             <div className={displayInModal ? '' : 'be bdl-UnifiedShareForm'}>
                 <LoadingIndicatorWrapper isLoading={isFetching} hideContent>
                     {!hasExpandedSections && allShareRestrictionWarning}
+                    {upsellInlineNotice && <div className="upsell-inline-notice">{upsellInlineNotice}</div>}
```

Comment:
I think we need to add a unit test for the use case that the `upsellInlineNotice` is not passed in and that it doesnt add the div into the DOM